hs_core/templates/pages/genericresource.html

```diff
@@ -476,6 +459,33 @@ <h4 class="modal-title">Confirm files deletion</h4>
                             {% endif %}
                         </div>
 
+                        {% if page.perms.change and metadata_form and cm.get_supported_upload_file_types != "()" %}
```

Comment:
I am not sure why ```cm.get_supported_upload_file_types != "()"``` is needed in the condition check. For the generic resource type resource, this condition should be false, right? Does that mean upload and download messages will not show for generic resource type resources? My understanding may not be right, but needs a bit of clarification here.